pkg/swiftring/job.go

```diff
@@ -42,6 +42,15 @@ func GetRingJob(instance *swiftv1beta1.SwiftRing, labels map[string]string) *bat
 	envVars["OWNER_UID"] = env.SetValue(string(instance.ObjectMeta.UID))
 	envVars["OWNER_NAME"] = env.SetValue(instance.ObjectMeta.Name)
 
+	volumes := getRingVolumes(instance)
+	volumeMounts := getRingVolumeMounts()
+
+	// add CA cert if defined
+	if instance.Spec.TLS.CaBundleSecretName != "" {
```

Comment:
swiftring controller should have an rbac for secrets

Reply:
Right, adding the RBAC rule